packages/front-end/services/tags.tsx

```diff
@@ -57,6 +57,38 @@ export const TAG_COLORS_MAP: Record<NonNullable<RadixColor>, string> = {
   sky: sky.sky11,
 };
 
+export const TAG_COLORS_BACKGROUND_MAP: Record<
```

Comment:
These big arrays look like a code smell to me. Naive question, but could we just do something like this...

```
import * as radixColors from "@radix-ui/colors";
...
const color = `${radixColors[c]}11`;
const bgcolor = `${radixColors[c]}5`;
```

Reply:
Oooh yea - that's slick. Thanks for the suggestion!

Reply:
Another approach (more for knowledge sharing, not necessarily a request for change) would be to use `background-color: var(--accent-5)` in CSS.

And the accent color can be controlled by the parent `data-accent-color` attribute like `data-accent-color="amber"`.

This is how the Radix components can update their colors dynamically based on the `color` prop that we pass in.

Reply:
Thanks, @ahdriel - that feels a bit more readable. I ultimately landed on a hybrid, the syntax below maps to the collect accent color.

```ts
const color = findClosestRadixColor(data.color) || "#029dd1";
backgroundColor: `var(--${color}-a3)`
```